### ISSUE TITLE
Notification is now showing for all the users

### ISSUE THREAD

**Issue by vinaykumarkota** [sagiegurari/angular-web-notification#28]
Hi,
I have implemented your code for my angular js project. I'm showing a notification on a button click event and it's working. But it's showing only in my system but not for all my website users.

If any push notification came it should show for all the website users right?. Please share your inputs to achieve that.

**Comment by sagiegurari** [comment]
maybe they didnt give permission to the website to allow for web notifications in the permissions popup

**Comment by sagiegurari** [comment]
Sorry, I don't understand what you mean.
Does it work for some users and for some it does not?
does it work for all users in 1 app, but not in another app?
Can you please explain it more detailed?

**Comment by vinaykumarkota** [comment]
I implemented push notifications on a button click event in one of my screen.So when I clicked the button on the website I'm getting push notifications but those notifications are not popping for others users of the same website.

When another user is clicking on the same button in his/her system again notifications are popping in his/her system but those notifications are not popping in my system.

I mean to say that push notifications are working in the local system only not for all the remaining users.
